dipy/tracking/tests/test_streamline.py

```diff
@@ -550,7 +550,8 @@ def test_deform_streamlines():
                                                   stream2world)
 
     # Subtract disps from new_streamlines in world space
-    orig_streamlines_world = list(np.subtract(new_streamlines_world, disps))
+    orig_streamlines_world = list(np.subtract(new_streamlines_world, disps,
+                                              dtype=list))
```

Comment:
I do not understand why `dtype=list` and not `dtype=object`?  Can you tell me more?

Reply:
If I read well, both operands to `subtract` are lists in this case, and `subtract` takes array-like arguments, which includes lists.

Reply:
- `new_streamlines_world` is a `Streamlines` object / class.  it can be a list of `ndarray` from the old API but not recommended
- `np.subtract` returns a `ndarray`
- `disps` is a `ndarray` or a list

it is quite generic so my preference is for `object`. Any other opinions are welcome!

Reply:
OK. Agreed Serge. Push forced in 58c0caa.